src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/InterceptorConfig.java

```diff
@@ -25,14 +25,16 @@ public class InterceptorConfig implements WebMvcConfigurer {
     static final String DETAILS = "/private/**/details";
     static final String MANAGING_OFFICERS_PRIVATE_DATA = "/private/**/managing-officers";
     static final String BENEFICIAL_OWNERS_PRIVATE_DATA = "/private/**/beneficial-owners";
-    static final String TRUST_PRIVATE_DATA = "/private/**/trusts/**";
+    static final String TRUST_DETAILS_PRIVATE_DATA = "/private/**/trusts/details";
+    static final String CORPORATE_TRUSTEES_PRIVATE_DATA = "/private/**/trusts/**/corporate-trustees";
```

Comment:
the second ** could be a single *
or
`static final String CORPORATE_TRUSTEES_PRIVATE_DATA = "/private/**/corporate-trustees";`

Reply:
Maybe take out line 28 and rely on line 25 matching. Possibly matching two paths causes an error?

Reply:
That could be the case

When we were testing putting "details" in place of the Trust ID made it work